test-logging.init.gradle.kts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Gradle Init Script for neotest-kotlin
+ *
+ * This helps force standardized and usable output for tests
+ * so that the plugin can effectively parse the output and it's usable
+ * for users.
+ */
+import org.gradle.api.tasks.testing.logging.TestExceptionFormat
+import org.gradle.api.tasks.testing.logging.TestLogEvent
+
+allprojects {
+    afterEvaluate {
+        tasks.withType<Test>().configureEach {
+            /**
+             * Force re-run the tests so we have output to parse
+             * [docs](https://blog.gradle.org/stop-rerunning-tests)
+             */
+            outputs.upToDateWhen { false }
```

Comment:
It's advised in the article I linked as "docs" to do this, but we need the output every time; otherwise, there's nothing for us to parse